StripeFinancialConnections/StripeFinancialConnections/Source/Native/InstitutionPicker/InstitutionTableView.swift

```diff
@@ -70,25 +75,35 @@ final class InstitutionTableView: UIView {
         )
         return manualEntryTableFooterView
     }()
-    private lazy var searchMoreBanksTableFooterView: InstitutionTableFooterView = {
-        let manualEntryTableFooterView = InstitutionTableFooterView(
-            title: STPLocalizedString(
-                "Search for more banks",
-                "The title of a button that appears at the bottom of a list of banks. The purpose of the button is to give users the option to search for more banks than we feature in the initial list of banks (where only the most popular ones will appear)."
-            ),
-            subtitle: nil,
-            image: .search,
-            didSelect: { [weak self] in
-                guard let self = self else { return }
-                self.delegate?.institutionTableViewDidSelectSearchForMoreBanks(self)
-            }
-        )
-        return manualEntryTableFooterView
+    private lazy var searchMoreBanksTableFooterView: InstitutionTableFooterView? = {
+        if institutionSearchDisabled {
+            return nil
+        } else {
```

Comment:
this fixes a separate issue where we did not handle `institutionSearchDisabled` in terms of not displaying the "searchMoreBanksTableFooterView" (a button that asks user if they want to search)